projects/helper/coreAssets.json

```diff
@@ -52,7 +52,8 @@
     "USDM": "0x59D9356E565Ab3A36dD77763Fc0d87fEaf85508C",
     "PRISMA": "0xdA47862a83dac0c112BA89c6abC2159b95afd71C",
     "FXN": "0x365accfca291e7d3914637abf1f7635db165bb09",
-    "USDe": "0x4c9edd5852cd905f086c759e8383e09bff1e68b3"
+    "USDe": "0x4c9edd5852cd905f086c759e8383e09bff1e68b3",
+    "DEUSD": "0x15700b564ca08d9439c58ca5053166e8317aa138"
```

Comment:
sorry, please remove the entry from here, I would like to keep this list as small as possible

Reply:
what is the criteria for this list? deUSD is bigger than some of the other stablecoins there

Reply:
this file is confused as an address book, but tokens here are used for pricing other tokens, so I would like to keep that set minimal.

There is no specific criteria, when I get some time, I will  likego through the code and other tokens from this list, but since other parts of code might reference it, I am scared to do it atm.